PoissonRegression.ipynb

```diff
@@ -0,0 +1,5371 @@
+{
```

Comment:
<p>Indexing the different categories of people like this (`df[1:1000,1]`) is pretty clunky. I would recommend sorting by indicator variable (whether they had a 0 for alcohol and meds, etc).</p><p><br></p><p>You are also contaminating your samples here: `df[1:1000,1]` and `df[1000:2000,1]` both share sample `1000`. It's the same for the others as well.</p>

 _Reply via <a href='https://app.reviewnb.com/TuringLang/TuringTutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Changed. Used indicator variable to get the data for plotting. For the first line, I used:</p><p>p1 = Plots.histogram(df[(df[:alcohol_taken] .== 0) .&amp; (df[:nomeds_taken] .== 0), 1], title = "no_alcohol+meds")&nbsp;&nbsp;</p><p><br></p>